docs/source/security.rst

```diff
@@ -0,0 +1,31 @@
+Security
+========
+
+Here at StackStorm we take security were seriously. If you believe you found a security issue or a
+vulnerability, please report it to us using one of the methods described below.
+
+Reporting a Vulnerability
+-------------------------
+
+.. note::
+
+   Please do not report security issues using our public Github repository or Slack instance. Use
+   the private mailing list described bellow.
+
+If you believe you found a security issue or a vulnerability, please send a description of it to
+our private mailing list at info [at] stackstorm [dot] com
+
+You are also encouraged to encrypt this email using PGP. Keys of our developers can be found at
+https://raw.githubusercontent.com/StackStorm/st2/master/KEYS.
```

Comment:
This link appears to be broken, i got a "404: not found" when visiting it.

Reply:
Yeah, will add a file shortly, still collecting the keys from our developers :)